.style-and-markdown-build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+
+ret=0
+
+cd "$(dirname $(realpath -s $0))"
+
+if grep -rn $'\t' modules assemblies pom.xml --include=pom.xml; then
+  echo "Tabs found!"
+  ret=1
+fi
+if grep -rn ' $' modules assemblies pom.xml --include=pom.xml; then
+  echo "Trailing spaces found!"
+  ret=1
+fi
+if grep -rn '	' etc; then
```

Comment:
Just to keep to one style:
```suggestion
if grep -rn $'\t' etc; then
```